src/styles/blocks/calendar.scss

```
@@ -0,0 +1,50 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+
+  width: containerWidth($squareSide, $gap, $containerPadding, $elementsInRow);
+
+  margin: 50px auto;
+  padding: $containerPadding;
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    width: $squareSide;
+    height: $squareSide;
+    border: $border solid black;
+    background-color: #eee;
+    font-family: Arial, sans-serif;
+    font-size: 30px;
+    transition: 0.5s;
```

Specify what will be in trasition:
```suggestion
    transition: transform 0.5s;
```